### PR TITLE
GUAC-1469 Update missing french translation

### PR DESCRIPTION
**Pull request by dynamiccast**
Some translations were missing because en.json changed since last update

### CODE REVIEW

**Comment by IT-Experte** [comment]
In line 624 a comma is missing. wrong json file format.

**Comment by dynamiccast** [comment]
Thank you for pointing that out. I just fixed it.

File: guacamole/src/main/webapp/translations/fr.json [hunk 1]
```diff
@@ -130,8 +132,15 @@
 
     },
 
+    "DATA_SOURCE_DEFAULT" : {
+        "NAME" : "Standard (XML)"
+    },
+
     "FORM" : {
 
+        "FIELD_PLACEHOLDER_DATE" : "DD-MM-YYYY",
```
Comment (mike-jumper) [inline guacamole/src/main/webapp/translations/fr.json]: This still needs to be "YYYY-MM-DD", or the French-language equivalent. This is not a date format string which affects display, but rather the format that the Guacamole webapp expects the user to conform to. If French uses different abbreviations for this pattern, then that is what would be translated here, but the placeholder still needs to guide the user to enter the year, month, and day, in that order, separated by dashes.

Comment (dynamiccast) [inline guacamole/src/main/webapp/translations/fr.json]: Ok got it. I though this was some sort of mask for the date. I just pushed an update.

**Pull request merged**